### PR TITLE
Small cards

### PR DESCRIPTION
**Pull request by JohnBinning**
## What's this PR do?

I wanted to make my cards smaller, I added a button to toggle/hide the bottom half

## Pre-Merge TODOs
 - [ x] Screenshots Included
 - [ x] Tests Added
  - I updated tests to account for having more conditional rendering
   - ANY change that can be tested must include testing
 - [ ] 2 PR reviews completed

## Where should the reviewer start?

Take a look at the ListItem component

## Screenshots

Before
screen shot 2017-06-05 at 5 56 23 pm

After
screen shot 2017-06-05 at 9 03 05 pm

record it http://recordit.co/MwG1rZ4wnn

### CODE REVIEW

**Comment by letakeane** [comment]
That is a slick interface, and the code looks super clean. Nice tests!

**Comment by jbevis** [comment]
I agree, UI looks great, and nice job getting that drop down incorporated with the tests to boot. Nice job John.

**Comment by JohnBinning** [comment]
@joshuajhun, @Tman22, or @rrgayhart
Have had two people give reviews.

**Comment by rrgayhart** [comment]
BTW you can embed recordits by getting the `gif` link -- it would look like this

```
![gif of site changes](http://g.recordit.co/MwG1rZ4wnn.gif)
```

File: app/assets/Components/ListItem/ListItem.js [hunk 1]
```diff
@@ -16,7 +16,50 @@ class ListItem extends Component {
...
+  }
+
+  toggleLowerCard() {
+    const newExpandedStatus = !this.state.expanded
+    this.setState({
+      expanded: newExpandedStatus
+    })
+  }
+
+  displayLowerCard(completedClass, completedText) {
```
Comment (rrgayhart) [inline app/assets/Components/ListItem/ListItem.js:31]: I think you could break this into its own component - since you already have it in a named function

**Pull request merged**